### DISCUSSION TITLE
Multi-selection form

### DISCUSSION THREAD

**Discussion by ghost** [streetcomplete/StreetComplete#5646] [Q&A] [answered]
Hey! quick question for a future PR, do we already have a quest with a multi-selection choice (like a checklist)? I've searched but found nothing, so I thought I'd ask just in case ^^.

Like this:
- [ ] Choice 1
- [ ] Choice 2
etc...

**Comment by matkoniecz** [comment, marked as answer]
Recycling and AFAIK sport on pitches allows multiselection, with image tapping.
